core/services/synchronization/telem/telem.proto

```diff
@@ -41,6 +41,7 @@ message TelemEnhancedEA {
     string feed=11;
     string chain_id=12;
     int64 observation=13;
-    int64 round=14;
-    int64 epoch=15;
+    string config_digest = 14;
+    int64 round=15;
+    int64 epoch=16;
```

Comment:
Is the last version already in the wild? Normally proto fields are append-only:
```suggestion
    int64 round=14;
    int64 epoch=15;
    string config_digest = 16;
```

Reply:
It's not in the wild yet, shifting the fields will not break any compatibility